client/src/pages/Profile.js

```diff
@@ -90,13 +117,24 @@ class ProfilePage extends Component {
   };
 
   render() {
+    const { classes } = this.props;
+    const { projects, investments } = this.state.profile;
+    const investedProjects = investments ? investments.map(investment => investment.project) : [];
+
     return (
-      <div className="profilePage">
+      <main className={classes.pageContent}>
         {this.renderUserInfo()}
-        {this.renderProjects()}
-      </div>
+        <div className={classes.projectInvestmentContent}>
+          <div className={classes.headerContent}>
+            <Typography classes={{ h2: classes.header }} variant="h2" onClick={() => this.changeDisplay('projects')}>Projects</Typography>
+            <Typography classes={{ h2: classes.header }} variant="h2" onClick={() => this.changeDisplay('investments')}>Investments</Typography>
```

Comment:
You can probably just do `className={classes.header}` instead, as much as I want to avoid individually editing each h2

Reply:
I've changed it now